stdlib/http/src/main/java/org/ballerinalang/net/http/HttpResourceDataElement.java

```diff
@@ -190,16 +190,16 @@ private String extractContentMediaType(String header) {
         return header;
     }
 
-    public HttpResource validateProduces(HttpResource resource, HttpCarbonMessage cMsg) {
+    private void validateProduces(HttpResource resource, HttpCarbonMessage cMsg) {
```

Comment:
Shall we make the return type as boolean and return false instead of throwing an exception for validation failure